.travis.yml

```diff
@@ -27,13 +27,13 @@ before_install:
     - pip install numpydoc
 
 install:
-    - pip install -e .[display]
+    - pip install -e .[display,testing]
 
 before_script:
     - pep8 mir_eval evaluators tests
 
 script:
-    - nosetests -v --with-coverage --cover-package=mir_eval
+    - nosetests -v --with-coverage --cover-package=mir_eval -w tests
```

Comment:
So the `-w tests` tells `nosetests` it's running within the `tests` folder, and this is why you removed the `tests` prefix from all the paths?